packages/ERTP/core/mint.js

```diff
@@ -147,17 +174,7 @@ allegedName must be truthy: ${allegedName}`;
         return unitOps.with(soFar, paymentKeeper.getUnits(payment));
       }, unitOps.empty());
 
-      const combinedPayment = harden({
-        getAssay() {
-          return assay;
-        },
-        getBalance() {
-          return paymentKeeper.getUnits(combinedPayment);
-        },
-        getName() {
-          return name;
-        },
-      });
+      const combinedPayment = makePayment(name);
```

Comment:
This looks like a big semantic change. It looks right. If it is right, the old code is very wrong: the old combinedPayment was not being customized by the config. They can't both be right. Do we have any tests that fail for the old one?

Reply:
Indeed. See https://github.com/Agoric/agoric-sdk/issues/390 for context on this change

> Do we have any tests that fail for the old one?

I didn't write one. I can do it

Reply:
That'd be great. I think we would just need to take one of the combine tests and have it test a simple non-fungible token.